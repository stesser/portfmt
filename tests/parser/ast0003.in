.if 1
fo=
.elifndef 2
.endif
