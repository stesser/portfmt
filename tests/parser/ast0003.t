.if 1
fo=
.elifndef 2
.endif
<<<<<<<<<
IF/IF :line 1 :indent 0 :test [1]["1"] :elseif 0
=> if:
	VARIABLE :line 2 :name "fo" :modifier =
=> else:
	IF/NDEF :line 3 :indent 0 :test [1]["2"] :elseif 1
