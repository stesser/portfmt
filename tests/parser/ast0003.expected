{ IF/IF, line 1, .indent = 0, .test = { 1 }, .elseif = 0 }
=> if:
	{ VARIABLE, line 2, .name = fo, .modifier = =, .words = {  } }
=> else:
	{ IF/NDEF, line 3, .indent = 0, .test = { 2 }, .elseif = 1 }
